extensions/objects/templates/math-expression-content-editor.directive.ts

```diff
@@ -16,14 +16,19 @@
  * @fileoverview Directive for math expression content editor.
  */
 
+require('mathjaxConfig.ts');
 require('directives/mathjax-bind.directive.ts');
+require('services/image-upload-helper.service.ts');
+require('services/alerts.service.ts');
+
 
 // Every editor directive should implement an alwaysEditable option. There
 // may be additional customization options for the editor that should be passed
 // in via initArgs.
 
 angular.module('oppia').directive('mathExpressionContentEditor', [
-  function() {
+  'AlertsService', 'ImageUploadHelperService',
+  function(AlertsService, ImageUploadHelperService) {
```

Comment:
Shouldn't localStorage be handled here in case that is required?

Reply:
In this directive, we don't save any images to the backend or localStorage. All this directive does is generate an SVG string for an equation pass it to the `rte-helper-modal-controller.ts`, the saving to the backend or localStorage happens in the rte controller.(I've added comments to explain the reason)

Reply:
Ah ok